Sources/TensorFlow/Operators/Math.swift

```diff
@@ -2158,6 +2158,36 @@ internal func _vjpMatmul<Scalar: TensorFlowFloatingPoint>(
     })
 }
 
+public extension Tensor where Scalar: TensorFlowFloatingPoint {
+    /// Computes the QR decomposition of each inner matrix in `tensor` such that
+    /// `tensor[..., :, :] = q[..., :, :] * r[..., :,:])`
```

Comment:
The syntax appears to be Python, and thus could be confusing to the user. We should use Swift code in all code examples. However, our APIs don't fully support this concise indexing style. For now, I believe it is more clear to use an English sentence to describe QR decomposition. This way you can also talk about the matrix properties of the results like orthogonality.

> Returns the QR decomposition of each inner matrix in the tensor, a tensor with inner orthogonal matrices `q` and a tensor with inner upper triangular matrices `r`, such that the tensor is equal to `matmul(q, r)`.